src/classes/CW_AudienceGroupCompControler.cls

```diff
@@ -0,0 +1,556 @@
+/**
+* File Name   :    CW_AudienceGroupCompControler.cls
+* Description :    B-110228/B-110230/B-109518
+*                  CIM, BI, Marketer must be able to View,Edit,Delete audience groups 
+*				   that are linked to the campaign 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 01/05/2014 Alex Panka		Intial - Partial copy of CW_AudienceGroupControler 
+* 0.2 03/05/2014 Alex Panka   	Revrite some functionality - (Best Practice)
+* 0.3 20/08/2014 Cristina P		Geeting members number for each group
+*/
+//Cristina P: 25/06/14: Changing the class from "with sharing" to "without sharing" 
+//so the user can create related records while cost/brief is in approval 
+public without sharing class CW_AudienceGroupCompControler {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.